nussknacker-dist/src/universal/bin/docker-run.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
```

Comment:
I don't think this file should be in our dist

Reply:
Now this is nussknacker-entrypoint and this file must be in our repo - it's an antry for docker.